qml/About.qml

```diff
@@ -78,7 +76,7 @@ Page {
             Label {
                textFormat: Text.RichText
                font.underline: false
-               text: i18n.tr("Released under the terms of the GNU GPL v3.<br>Source code available on") + " <a style=\"text-decoration: none;color:"+about_column.themableBlue+";\" href=\"https://github.com/ernesst/ActivityTracker\">GitHub.com</a>"
+               text: i18n.tr("Released under the terms of the GNU GPL v3.<br>Source code available on") + " <a "+about_column.linkColor+" style=\"text-decoration: none;\" href=\"https://github.com/ernesst/ActivityTracker\">GitHub.com</a>"
```

Comment:
Sorry if I'm nitpicking this too much, but is it fine to put the style attribute twice here? Did you tested it and it works?

Reply:
1. I did not add this, that is from the original code
2. just tried it, seems to work, results in the github link not having a underline
3. Since all the other links do have an underline, lets remove it here too to have a consistent layout

Reply:
1. The original code had the `style` attribute only once in that line and had the `color` property explicit and just the color appended via `about_column.themableBlue` variable, while you used `about_column.linkColor` which brings in the full html style attribute with color property set.
2. 👍🏻 
3. Oh, ok not sure why I did this, let's make it consistent.

Anyway, this LGTM